### PR TITLE
185 3.23 application checking page

### PR DESCRIPTION
**Pull request by aloeser**
Add a application checking page, so participants can check their applications after creating/editing them

### CODE REVIEW

**Comment by bjrne** [comment]
@tobiduer is this enough for #123 ?

**Comment by tobiduer** [comment]
@bjrne: Yes. I closed the other issue.

**Review by querenker — CHANGES_REQUESTED** [review]
I especially like the use of the different tests 👍

File: app/views/application_letters/check.html.erb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    end
+    if @application_letter.vegan
+      habits.push(ApplicationLetter.human_attribute_name(:vegan))
+    end
+    if @application_letter.allergic
+      habits.push(ApplicationLetter.human_attribute_name(:allergic))
+    end
+  %>
+  <strong><%=t('.eating_habits') %>:</strong>
+  <%= habits.join(', ') %><br/>
```
Comment (aloeser) [inline app/views/application_letters/check.html.erb]: We did ask him, and he said this behaviour is exactly what is wanted

File: app/views/application_letters/check.html.erb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    if @application_letter.vegeterian
+      habits.push(ApplicationLetter.human_attribute_name(:vegeterian))
+    end
+    if @application_letter.vegan
+      habits.push(ApplicationLetter.human_attribute_name(:vegan))
+    end
+    if @application_letter.allergic
+      habits.push(ApplicationLetter.human_attribute_name(:allergic))
+    end
+  %>
```
Comment (aloeser) [inline app/views/application_letters/check.html.erb]: So you would prefer to put do it in the controller and pass it as @habits? to the view?

Comment (querenker) [inline app/views/application_letters/check.html.erb]: Maybe a model method for the application_letter that returns the habits as a list?

**Comment by aloeser** [comment]
@tobiduer I dont think that #123 is completely done with this story. #123 also requires to replace the age-Field by birthday, which is not (yet) part of this user story.

**Comment by tobiduer** [comment]
@aloeser: OK than please show the birthday instead of the age.

**Review by ekrebs5 — APPROVED** [review]

**Review by timbasel — APPROVED** [review]

**Review by querenker — APPROVED** [review]

**Pull request merged**